src/stars.css

```diff
@@ -0,0 +1,20 @@
+.stars-review {
+  display: flex;
+}
```

Comment:
```suggestion
.stars-review {
  display: flex;
  background-image: url(images/star.svg);
}
```
Add inactive stars
![image](https://user-images.githubusercontent.com/53319174/180009516-343dec3b-5f62-4152-a05f-19da5e5a3b50.png)